tests/test_manager.rs

```diff
@@ -33,7 +34,8 @@ fn test_database_add_remove_script() {
     db.remove_all().unwrap();
 
     let name = Id::<ScriptId>::new("Sample Ruleset");
-    db.put(&name, &load_json("./examples/ruleset.json")).unwrap();
+    db.put(&name, &load_json("./examples/ruleset.json"), &User::Id(1)).unwrap();
```

Comment:
Ideally, we would like a test that ensures that the user id sent to the destination channel is correct. I realize that this is more complex, but can you 

1/ Test it manually to confirm that it works;
2/ File a followup bug
?